hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/DataXceiver.java

```diff
@@ -587,7 +587,7 @@ public void readBlock(final ExtendedBlock block,
     final String clientTraceFmt =
       clientName.length() > 0 && ClientTraceLog.isInfoEnabled()
         ? String.format(DN_CLIENTTRACE_FORMAT, localAddress, remoteAddress,
-            "%d", "HDFS_READ", clientName, "%d",
+            "", "%d", "HDFS_READ", clientName, "%d",
```

Comment:
looks like redundant change?

Reply:
Because volume has been added to DN_CLIENTTRACE_FORMAT, some adaptations have been made.